### PR TITLE
Keep compareForm open (Forms/Showhistory)

### PR DESCRIPTION
**Pull request by xylographe**

### CODE REVIEW

File: src/Forms/ShowHistory.cs [hunk 1]
```diff
@@ -113,7 +113,7 @@ private void ButtonCompareClick(object sender, EventArgs e)
                 using (var compareForm = new Compare())
                 {
                     compareForm.Initialize(_subtitle, Configuration.Settings.Language.General.CurrentSubtitle, h2.Subtitle, descr2);
-                    compareForm.Show();
+                    compareForm.ShowDialog();
```
Comment (niksedk) [inline src/Forms/ShowHistory.cs]: I prefer compareForm.ShowDialog(this); (windows will be linked... works better if something is minimized)

**Comment by xylographe** [comment]
Updated (ShowDialog(*this*)).

**Comment by niksedk** [comment]
Good bug fix :)

File: src/Forms/ShowHistory.cs (hunk 1 above)
Comment (xylographe) [inline src/Forms/ShowHistory.cs]: After re-reading docu I experimented a bit. Makes quite a difference.
Thank you, for the tip.

**Pull request merged**